src/lib/containers/dataaccess/SubmissionPage.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
```

Comment:
I like that you are defining the msw responses right in the story itself, so it's clear where this data is coming from!